js/statements/createmacro.js

```diff
@@ -0,0 +1,29 @@
+
+function GenerateCreateMacro(options = {}) {
+	return Diagram([
+		AutomaticStack([
+			Keyword("CREATE"),
+			Choice(0, [
+				Keyword("MACRO"),
+				Keyword("FUNCTION"),
+				]
+			),
+			Keyword("AS"),
```

Comment:
This seems to miss the parameter list and the name of the function, no?
Probably want to add something like:

```js
Expression("macro-name"),
Keyword("("),
OneOrMore(Sequence([
  Expression("param-name"),
  Optional(Sequence([
    Keyword("="),
    Expression("default-value")
  ], "skip"),
]), Keyword(",")),
Keyword(")")
```

Reply:
Good point, I was too busy trying to figure out how the railroad works that I forgot the function name completely

Reply:
I did something like this, but where normal params must come before default params